css/events.css

```diff
@@ -161,6 +161,28 @@ Section Styles (background, padding, etc.)
 	margin: 0 auto;
 }
 
+.event-detail-container {
```

Comment:
Since we have events page + single event page styles in one file, can you leave a comment to separate the styles?